core/src/main/java/io/grpc/InternalServerInterceptors.java

```diff
@@ -26,6 +26,12 @@
     return ServerInterceptors.InterceptCallHandler.create(interceptor, callHandler);
   }
 
+  public static <OReqT, ORespT, WReqT, WRespT> ServerMethodDefinition<WReqT, WRespT> wrapMethod(
```

Comment:
It looks like `?` would also work here.

Reply:
I think we can remove it from this accessor class.